src/material/button/_button-theme.scss

```diff
@@ -229,8 +228,21 @@
 }
 
 @mixin typography($theme) {
-  @include mdc-helpers.using-mdc-typography($theme) {
```

Comment:
We have a similar usage to this one in the FAB theme. I suspect it's a noop so should we either migrate it or delete it?